src/core/panel/plugin-panel.tsx

```diff
@@ -48,10 +48,10 @@ export abstract class UIPPluginPanel extends UIPPlugin {
     const a11yLabel = this.collapsible ? 'Collapse/expand' + this.label : this.label;
     const hasToolbar = this.$toolbar?.children.length;
     return (
-      <div class={type.is + '-header uip-plugin-header' + (this.label ? '' : ' no-label') + (hasToolbar ? ' has-toolbar' : '')}>
-        {this.$icon ? <span class="uip-plugin-header-icon" title={this.label}>{this.$icon}</span> : ''}
-        <span class="uip-plugin-header-title">{this.label}</span>
-        {this.collapsible ? <button type="button" class="uip-plugin-header-trigger" aria-label={a11yLabel} title={a11yLabel}/> : ''}
+      <div className={type.is + '-header uip-plugin-header' + (this.label ? '' : ' no-label') + (hasToolbar ? ' has-toolbar' : '')}>
+        {this.$icon ? <span className='uip-plugin-header-icon' title={this.label}>{this.$icon}</span> : ''}
+        <span class={`uip-plugin-header-title ${type.is}-title`}>{this.label}</span>
```

Comment:
```suggestion
        <span className={`uip-plugin-header-title ${type.is}-title`}>{this.label}</span>
```

Reply:
Yep, thanks, we planned a cleanup PR with that update so temporrarary skipping